internal/tiltfile/tiltfile.go

```diff
@@ -3,8 +3,10 @@ package tiltfile
 import (
 	"fmt"
 	"github.com/google/skylark"
+	"github.com/pkg/errors"
```

Comment:
did you intend to use plain "errors" from the standard lib?

Reply:
is it idiomatic to use fmt.Errorf on bare (non-format) strings?